rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Check rabbit cluster status
+  shell: "rabbitmqctl cluster_status"
+  register: rabbitmqctl_result
+  delegate_to: "{{ groups['rabbitmq_all'][0] }}"
```

Comment:
IIRC a step in the maint plan is to disable MaaS alerting before embarking on the upgrade. I do agree that a step should be to observe the dashboard or something, but I don't see why we should not explicitly do our own checks to verify that things we need to be working are working.